glotaran/deprecation/modules/test/test_parameter_parameter_group.py

```diff
@@ -15,11 +15,11 @@ def test_parameter_group_to_csv_no_stderr(tmp_path: Path):
     expected = dedent(
         """\
         label,value,expression,minimum,maximum,non-negative,vary,standard-error
-        rates.species_1,0.5,None,-inf,inf,False,True,None
-        rates.species_2,0.3,None,-inf,inf,False,True,None
-        rates.species_3,0.1,None,-inf,inf,False,True,None
-        irf.center,0.3,None,-inf,inf,False,True,None
-        irf.width,0.1,None,-inf,inf,False,True,None
+        rates.species_1,0.5,None,,,False,True,None
```

Comment:
@s-weigand Does this correctly translate to `(-)np.inf`? it is important to have them lie this in optimization if no value is provided.

Reply:
Yap it is tested in `glotaran/utils/test/test_io.py`. 
But maybe we should make dropping the `inf` values optional for text-based formats (`csv`, `tsv`)?

Reply:
Also `tsv` and `csv` aren't consistent ... in that 😖

Reply:
Fixed consistency issue and added test for mixed loading in 37bb74a

Reply:
Restored original test 9602f53, so now the deprecate method behaves exactly like before